app/src/main/kotlin/org/fossify/contacts/dialogs/ExportContactsDialog.kt

```diff
@@ -28,7 +28,7 @@ class ExportContactsDialog(
     init {
         val binding = DialogExportContactsBinding.inflate(activity.layoutInflater).apply {
             exportContactsFolder.setText(activity.humanizePath(realPath))
-            exportContactsFilename.setText("contacts_${activity.getCurrentFormattedDateTime()}")
+            exportContactsFilename.setText("Contacts_${activity.getCurrentFormattedDateTime()}")
```

Comment:
`Contacts_` is open to suggestion, as these manual exports would have the same name as automatic backup files.
Besides, this might even conflict (although rare) if the user happens to export at the same time an automatic backup is being performed.

Reply:
`contacts_` is fine.